src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/handlers/presubmit/PaymentAppealHandler.java

```diff
@@ -71,7 +57,7 @@ public boolean canHandle(
         requireNonNull(callback, "callback must not be null");
 
         return (callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT)
-                && (callback.getEvent() == Event.PAYMENT_APPEAL);
+                && (callback.getEvent() == Event.PAY_AND_SUBMIT_APPEAL);
```

Comment:
I would just add `PAY_AND_SUBMIT_APPEAL` event and keep `PAYMENT_APPEAL` as it is. Eventually we need to display PBA account list for both events

Reply:
Done